src/test/resources/s3-valid-inbound-cases.xml

```diff
@@ -66,6 +66,22 @@
 		<integration:poller fixed-rate="1000"/>
 	</int-aws:s3-inbound-channel-adapter>
 
+	<int-aws:s3-inbound-channel-adapter
+			id="validInboundWithProxy"
+			bucket="TestBucket"
+			propertiesFile="testawscredentials.properties"
+			temporary-suffix=".temp"
+			local-directory="${foo.dir:/}"
+			remote-directory="remote"
+			accept-sub-folders="true"
+			max-objects-per-batch="100"
+			file-name-regex="[A-Za-z0-9]+\\.txt"
+			channel="inbound"
+			auto-startup="false"
+            proxyHost="localhost"
+            proxyPort="80">
```

Comment:
> If above is the case, then only modification would be to AbstractAmazonS3Operations to have a constructor to accept Client Configuration.

Yes, it is, but only for the target application not for the Framework. We really can just rely on the `AmazonS3Client` injection. That is another way to fix JIRA - rather than fix the fragile code, we just remove it and allow to use the Framework with more simple way. 

Yes, you can contribute this way, too, but I'm not sure even myself how it will go from there...
I'd like to start refactoring from the `AmazonS3MessageHandler`